scripts/image-builder/cleanup.sh

```diff
@@ -82,6 +82,7 @@ restart_osbuilder_services() {
     if ! systemctl is-active -q osbuild-composer.socket &>/dev/null ; then
         title "Starting osbuild services"
         sudo systemctl start osbuild-composer.socket
+        sleep 1 # Give composer some time to be ready for workers
```

Comment:
I'd prefer not to have sleep. Can we try starting workers 3 times in a loop with sleep in between?